File: apps/translate-tencent_HY-MT/Android/app/src/main/java/ai/zetic/demo/offlinetranslator/config/ZeticConfig.kt
package ai.zetic.demo.offlinetranslator.config

/**
 * ZETIC.ai Melange deployment config for the on-device translation model.
 *
 * Note: the iOS app uses the slug `vaibhav-zetic/tencent_HY-MT`; this Android build uses
 * `palm/tencent_HY-MT` per the provided Android deployment snippet (same dev key, version 1).
 * If the model fails to download on a real device, this slug is the first thing to verify.
 */
object ZeticConfig {
    const val personalKey = "YOUR_MLANGE_KEY"
    const val modelName = "palm/tencent_HY-MT"
    const val modelVersion = 1
}

File: apps/translate-tencent_HY-MT/iOS/OfflineTranslator/Config/ZeticConfig.swift
import Foundation

/// Configuration for the ZETIC.ai Melange on-device model.
///
/// `personalKey` is the placeholder Melange Personal Access Token — run
/// `./adapt_mlange_key.sh` from the repo root to fill it in. `modelName`/`version`
/// point at the Tencent HY-MT (Hunyuan-MT) model uploaded to the Melange dashboard.
enum ZeticConfig {
    static let personalKey = "YOUR_MLANGE_KEY"
    static let modelName = "vaibhav-zetic/tencent_HY-MT"
    static let modelVersion = 1
}
